docs/dev/marketplace_offering.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Build A Marketplace Offering
+type: documentation
+aliases: 
+- 'developers/marketplace/'
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/marketplace_offering.md"
+further_reading:
+- link: "https://www.datadoghq.com/partner/"
+  tag: "Partner Network"
+  text: "Datadog Partner Network"
+- link: "https://www.datadoghq.com/blog/datadog-marketplace/"
+  tag: "Blog"
+  text: "Expand your monitoring reach with the Datadog Marketplace"
+- link: "/developers/integrations/marketplace_offering/"
+  tag: "Documentation"
+  text: "Learn how to develop a Marketplace offering"
+- link: "/developers/datadog_apps/"
+  tag: "Documentation"
+  text: "Learn about Datadog Apps"
+---
+
+## Overview
+
+The Datadog Marketplace is a digital marketplace where Datadog Technology Partners can list their paid offerings to Datadog users.  
+
+While the **Integrations** page includes integrations built by both Datadog and Technology Partners at no cost, the **Marketplace**  is a commercial platform for Datadog customers and Technology Partners to buy and sell a variety of offerings including Agent-based or API-based integrations, software licenses, and professional services.
+
+## List an offering 
+
+All Technology Partners can list an out-of-the-box integration on the **Integrations** page, or a commercial paid offering on the **Marketplace** page. The following types of offerings are supported on the Datadog Marketplace:
```

Comment:
I don't think we need to call out the Integrations page here since this is specific to the Marketplace.